gps/index.html

```diff
@@ -10,27 +10,27 @@
 date: 2018-04-09
 categories: []
 ---
-<h2 class="text-left"><i class="fas fa-fw fa-question"></i> What is a Reference Network</h2>
+<h2 class="text-left"><i class="fas fa-fw fa-question"></i> What Is a Reference Network?</h2>
 <img class="inline-text-right outline" src="images/turn.png" alt="TURN GPS Network">
-<p>A GPS network is a high precision Global Navigation Satellite System (GNSS) providing real time corrections and data for post processing. A GNSS network consists of permanently located GPS receivers, installed across an area, that generate real time high accuracy GPS positioning. Basically, all of the permanently installed GPS receivers send their data to a central system via the internet, that analyzes the input and adjusts and corrects the data to provide the most accurate locations usually down to centimeter level. Our networks use the Trimble PIVOT VRS Platform. The generated data is sent through a wireless connection using the Networked Transport of RTCM via Internet Protocol (NTRIP).</p>
+<p>A GPS network is a high-precision Global Navigation Satellite System (GNSS) providing real-time corrections and data for post processing. A GNSS network consists of permanently located GPS receivers, installed across an area, that generate real-time, high-accuracy GPS positioning. Basically, all of the permanently installed GPS receivers send their data to a central system via the internet. This central system analyzes the input and adjusts and corrects the data to provide the most accurate locations, usually down to centimeter level. Our networks use the Trimble PIVOT VRS Platform. The generated data is sent through a wireless connection using the Networked Transport of RTCM via Internet Protocol (NTRIP).</p>
```

Comment:
GPS data coordinates are collected at physical locations. Since we have a network of receivers, the gps data coordinates are compared and corrected for the users virtual location. time and distance gps data is distorted by space and these networks aim to remove that as much as possible. 

Post processing would be how it was done before. A person would go out and survey a location, then come home and download the other locations data and post process to figure out where the location actually was. So the real time network actually replaces the need to post process. It does the post processing  part in real time.

Reply:
Ok, I trust you on this one! I'll just leave the language as is!

Reply:
I think we should make this more clear.